oresat_dxwifi/resources/temperature.py

```diff
@@ -75,13 +66,12 @@ def find_temperature(self) -> float:
         Returns:
             temperature: Calculated temperature value
         """
-        temperature = None
+        temperature = 0.0
```

Comment:
`0.0` isn't ideal default since that's a valid value. Previously the default was `None`, which indicated that the value was invalid or that we were unsuccessful in fetching it. We could make it something lower than absolute zero (-273 point something) or say to heck with it and leave it as `0.0`. We do log on error, at least.

Reply:
Sound good, but docstring/annotations do not say anything about returning a `None`.

Reply:
That was my bad. I should have included it in the doc string, but I think we could just go with this since it is physically impossible.
```suggestion
        temperature = -1000.0
```